.github/workflows/ci_cd.yml

```diff
@@ -822,12 +822,24 @@ jobs:
       - name: Run Playwright tests
         run: just frontend/run ${{ matrix.script }} --workers=2
 
+      - name: Check for changes after running tests with -u
+        id: check_snapshots_changed
+        run: |
+          # shellcheck disable=SC2086
+          if [ -n "$(git diff --binary --name-only)" ]; then
+            git add . && git diff --staged --binary | gzip -f > /tmp/snapshot_diff.patch.gz
+            echo "snapshots_changed=true" >> $GITHUB_OUTPUT
+            git diff --binary --name-only
+          else
+            echo "snapshots_changed=false" >> $GITHUB_OUTPUT
+          fi
```

Comment:
Ideally we wouldn't disable shell check rules unless absolutely necessary. Is there a reason why `$GITHUB_OUTPUT` cannot be double quoted? If so, can you add a comment explaining why we need to disable the check for this script?

Reply:
No reason at all. It didn't occur to me that that was the exact reason as I'd thought it had something to do with the compression command `gzip -f > /tmp/snapshot_diff.patch.gz`. Fixed that.